openreview/venue_request/venue_request.py

```diff
@@ -533,6 +548,13 @@ def setup_comment_stage(self):
                 'required': False,
                 'default': 'Yes, enable chat between committee members',
                 'order': 33
+            },
+            'comment_description': {
+                'order': 34,
+                'value-regex': '[\\S\\s]{1,5000}',
+                'description': 'Specify a description for the comment stage. This will be shown in the comment form.',
```

Comment:
Can we add to the descriptions `Markdown is supported`?

Also, I like the default instructions that we build for the comment stage when the description is empty, but PCs don't know what it is or that we prepopulate it. Should we convey that to them? I'm not sure what the best way would be.

Reply:
Sure, I can add it to the description. We can ask Xukun to render the markdown icon in the text area when the field has markdown: true.

I don't understand the last statement, do you think we shouldn't prepopulate the description for them?

Reply:
I think the description we build is really good, but it's not clear in the request form that if `comment_description` is empty then we create it. The PCs can't see what the default is, does that make sense?

I think what we normally do in this situation is add it to the docs and link the docs in the request form. We're adding a new field to the stages so maybe it's good to update the docs, unless there a simpler way we can inform PCs of the default value? What do you think @celestemartinez, do you think it's necessary ?

Reply:
We can add this default value to the docs, either in the comment default form or [the comment stage](https://docs.openreview.net/reference/stages/comment-stage). I don't think it is particularly necessary but I think it could be nice to have it somewhere. 

It's not easy to show the default value in the request form since this is dynamically created depending on the options the PC choose at the time of creating/updating the comment stage.

Reply:
ok, I can mention in the description of the comment stage that a description will be autogenerated if the value is blank.

For other stages, I don't need to say anything, the value is optional

Reply:
That works too!